docs/language/internals.rst

```diff
@@ -426,10 +429,10 @@ so our re-written ``nif`` would look like:
 
    (defmacro nif [expr pos-form zero-form neg-form]
```

Comment:
Yeah, looks like it. My bad. I must have done something extremely silly while testing that.